server/test/ping.spec.js

```diff
@@ -0,0 +1,247 @@
+const chai = require("chai");
+const chaiHttp = require("chai-http");
+
+const resetDB = require("../scripts/resetDB");
+const app = require("../app.js");
+const { expect } = require("chai");
+
+chai.should();
+chai.use(chaiHttp);
+
+describe("/GET health", () => {
+    it("it should return 200 if the server is running", (done) => {
+        chai.request(app)
+            .get("/health/")
+            .end((err, res) => {
+                err && console.error(err);
+                res.should.have.status(200);
+                res.body.should.have.property("success").eql(true);
+                done();
+            });
+    });
+});
+
+// delete everthing in DB to have consistant results in testing
```

Comment:
Nice job writing tests, but I don't think we want to have tests that delete our entire database. In general though, we want to focus on features over testing given the short time frame for the project, so don't spend too much time on tests.

Reply:
I delete everything to have blank slate for consistent tests.
I just updated the `npm test` to use a separate database for testing to avoid accidentally deleting everything from production database, by passing `ATLAS_URI` env var

```bash
"test": "NODE_ENV=test ATLAS_URI=mongodb://localhost:27017/corn-pops-test ./node_modules/.bin/mocha --exit --timeout 30000"
```

I completely agree, I only have few tests with happy path. It helps me be confident when I make changes that nothing brakes.